### ISSUE TITLE
Lichens of the Pacific Northwest (lichens-pnw)

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2-migration#727]
**Item Count:** 5,216 items

**Item Types:** Image

**Access Restrictions:** 0

**Complex Objects:** 0

**pid List:** https://github.com/OregonDigital/OD2-migration/blob/master/collname/pidlist

**Comment by carakey** [comment]
The collection looks good overall and is ready for bulk approve. I cleaned up the collection description slightly:

* Changed `&amp;` entity to `&`
* Added the list of common abbreviations; this was formatted as a `ul` in OD1 and was not present in OD2, but the description still included a reference to the list.

Some readability is lost for this collection description since we aren't using the HTML markup.

During QA I had one work get 'hung up' when it was reviewed: [df71wk33q](https://prod.oregondigital.org/concern/images/df71wk33q) - it showed with the Deposited flag on the work, but was not available from a non-authenticated browser and was not present when browsing the collection, and was not returned in search results in either front or back end. I sent it back through the review workflow and the issue was resolved.
